camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynComponentTemplateResolver.java

```diff
@@ -257,6 +257,7 @@ public boolean canResolve() {
         new BrooklynEntityDecorationResolver.EnricherSpecResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.InitializerResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.SpecParameterResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
+        new BrooklynEntityDecorationResolver.TagsResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
```

Comment:
Thanks @neykov - makes sense. However, a much bigger topic is how we really should be defining effectors/feeds in yaml. To me, the `brooklyn.initializers` feels like a powerful hack - re-using a generic feature to add things that are fundamental to blueprints. I think it's confusing for users to have to use `brooklyn.initializers` rather than us having first-class `brooklyn.effectors`. But let's not discuss that here - it can be a dev@brooklyn discussion when we're ready!